oldsite/static/js/toggle.js

```diff
@@ -0,0 +1,12 @@
+const currentPagePath = window.location.pathname;
+const currentPageName = currentPagePath.substring(currentPagePath.lastIndexOf('/') + 1);
+console.log(currentPageName);
```

Comment:
I would remove the `console.log()` before merging this.

Reply:
Doh!